boreas/CMakeLists.txt

```
@@ -62,6 +62,7 @@ if (PCAP_CONFIG)
   execute_process (COMMAND pcap-config --libs
     OUTPUT_VARIABLE PCAP_LDFLAGS
     OUTPUT_STRIP_TRAILING_WHITESPACE)
+  string(STRIP PCAP_LDFLAGS PCAP_LDFLAGS)
```

First argument for STRIP must be a string .
```suggestion
  string(STRIP "${PCAP_LDFLAGS}" PCAP_LDFLAGS)
```